content/c-sharp/concepts/strings/terms/indexof/indexof.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.IndexOf()'
+Description: 'Returns the index of a specified character or substring in a string.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Strings'
+  - 'String Searching'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`IndexOf()`** method is a string method that returns the index of the first occurrence of a specified character or substring in a given string. If the character or substring is not found, it returns `-1`.
+
+## Syntax
+
+```pseudo
+public int IndexOf(char value);
+public int IndexOf(string value);
+public int IndexOf(char value, int startIndex);
+public int IndexOf(string value, int startIndex);
+public int IndexOf(string value, int startIndex, int count);
+```
+
+- `value` : The substring or character to search for.
+- `startIndex` (optional): The index from where the search should start.
+- `count` (optional): Maximum number of characters to search.
+
+## Example
```

Comment:
For simple methods one example is usually sufficient. I think the second example is more comprehensive and interesting, so please remove the first example.